opentracing-spring-web/src/main/java/io/opentracing/contrib/spring/web/client/decorator/BaseRestTemplateSpanDecorator.java

```diff
@@ -0,0 +1,22 @@
+package io.opentracing.contrib.spring.web.client.decorator;
+
+import io.opentracing.Span;
+import io.opentracing.contrib.spring.web.client.RestTemplateSpanDecorator;
+import org.springframework.http.HttpRequest;
+import org.springframework.http.client.ClientHttpResponse;
+
+public class BaseRestTemplateSpanDecorator implements RestTemplateSpanDecorator {
```

Comment:
This can be achieved by standard spring filter/interceptor and getting active span. I would rather keep the APIs provided by instrumentations small.

Reply:
I'm not sure to follow, but if that helps the `ResttemplateSpanDecorator` interface is not new to this pull request, it was added here: https://github.com/opentracing-contrib/java-spring-web/blob/55cc1d7c9e8dc691d327022730e37f300e61e291/opentracing-spring-web/src/main/java/io/opentracing/contrib/spring/web/client/RestTemplateSpanDecorator.java